### PR TITLE
[script.module.chompjs@matrix] 1.0.0

### PR DESCRIPTION
**Pull request by joaopa00**
### Description
Add chomps as a new module for Kodi. Ease the conversion of javascript object into python object.
### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
Chompjs will be requested by the next version of Catchuptvandmore plugin.

### CODE REVIEW

File: script.module.chompjs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,190 @@
+# Prerequisites
```
Comment (basrieter) [inline script.module.chompjs/.gitignore:1]: Exclude plas

Comment (joaopa00) [reply script.module.chompjs/.gitignore]: Removed the useless file

**Comment by joaopa00** [comment]
As requested, I removed the problematic fle.

**Comment by basrieter** [comment]
> As requested, I removed the problematic fle.

And the comment about te compilation?

**Comment by basrieter** [comment]
> Sorry, I do not see what you mean. Can you explicit what you want me to modify?

image

The main `chompjs`-module needs to be compiled before you can use it. Compilation cannot be done within Kodi.

**Comment by joaopa00** [comment]
I did not find a way to compile automatically the c file for any platform. So I will close this PR.  Iwill have to find a way to avoid this library in Catchuptvandmore project.

**Pull request closed without merging**